core/src/main/java/me/deadlight/ezchestshop/data/Config.java

```diff
@@ -99,13 +99,19 @@ public class Config {
 
     public static boolean shopCommandsEnabled;
     public static ShopCommandManager shopCommandManager;
+    public static int taxesshopplayer;
+    public static int taxesplayershop;
+    public static String leader;
 
 
     public static void loadConfig() {
         //reloading config.yml
 
         EzChestShop.getPlugin().reloadConfig();
         FileConfiguration config = EzChestShop.getPlugin().getConfig();
+        taxesshopplayer = config.getInt("economy.taxes-shop-player");
+        taxesplayershop = config.getInt("economy.taxes-player-shop");
+        leader = config.getString("economy.leader");
```

Comment:
Is the leader the server owner who gets the taxes? I was expecting the money to disappear out of the economy. This would be more in line with our other features like admin shops. At least make this the default option, if you do require a leader. Also I am not sure if leader is descriptive enough again. Maybe try `economy.tax-collecting-player-name`?